packages/playwright/src/matchers/expect.ts

```diff
@@ -268,9 +269,13 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
           jestError.stack = jestError.name + ': ' + newMessage + '\n' + stringifyStackFrames(stackFrames).join('\n');
         }
 
-        const serializedError = serializeError(jestError);
-        // Serialized error has filtered stack trace.
-        jestError.stack = serializedError.stack;
+        // Use the exact stack that we entered the matcher with.
+        jestError.stack = jestError.name + ': ' + jestError.message + '\n' + stringifyStackFrames(stackFrames).join('\n');
```

Comment:
- This looks unrelated to the change 😄 
- It seems we can merge this with the line 269?